### DISCUSSION TITLE
Octernship eligiblity

### DISCUSSION THREAD

**Discussion by jai10-dev** [community/community#49468] [GitHub Education]
What I considered an active user for github? Also this is not my primary git account, I know github allows to add another email Id and make that primary. But I have github student(signed up for copilot and reactivated for octernship) through this , will I lost my student privileges if I merge this to my primary account now?

**Comment by samus-aran** [comment]
Hi @jai10-dev 

It’s not possible to transfer Global Campus benefits over from one GitHub account to another.

However, you are able to merge your accounts, but the account with the benefits would be the main account (not your other or you'll lose your student status). You can add your non-academic email from the previous account to the account containing the academic email. If you desire you can then rename this account once merged. For more details check out out the [merging of your personal accounts](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts) documentation.
